movies-search-engine/add_data.py

```diff
@@ -0,0 +1,144 @@
+#importing required libraries
+import weaviate
+import pandas as pd
+#setting up client
+client = weaviate.Client("http://localhost:8080")
+
+
+
+#  UNCOMMENT BELOW LINE IF THIS IS NOT THE FIRST TIME YOU ARE ADDING DATA FOR THIS SCHEMA
+# client.schema.delete_class('Movies')

```

Comment:
Can we make this conditional?
The script could check if the schema for `Movies` exists and if it does then delete it